cmd/save.go

```diff
@@ -184,19 +185,21 @@ continue?`, true) {
 		}
 	}
 
-	res := &reporef.DatasetRef{}
+	res := &dataset.Dataset{}
 	if err = o.DatasetMethods.Save(p, res); err != nil {
 		return err
 	}
 
 	o.StopSpinner()
-	printSuccess(o.ErrOut, "dataset saved: %s", res)
-	if res.Dataset.Structure != nil && res.Dataset.Structure.ErrCount > 0 {
-		printWarning(o.ErrOut, fmt.Sprintf("this dataset has %d validation errors", res.Dataset.Structure.ErrCount))
+	ref := dsref.ConvertDatasetToVersionInfo(res).SimpleRef()
+	ref.ProfileID = ""
```

Comment:
Why do we remove the profileID here?

Reply:
to make the call to `ref.String()` below not print such a verbose reference.

without profileID:
```
dataset saved rm_me_3/saxaphones@/ipfs/QmRL5Wq4K1JXYZsgZKWJpdFJvQEsNCc6n9twRkzSGsCLbF
```

with:
```
dataset saved  rm_me_3/saxaphones@QmcPmrQYqFamdHYFL8qbfpbT3dRNRVanLmFEY9srWsddt9/ipfs/QmRL5Wq4K1JXYZsgZKWJpdFJvQEsNCc6n9twRkzSGsCLbF
```

Long term it'd be better to add a method to `dsref` that does this